docs/pages/installation/amazon-ecs.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Installing Teleport on Amazon ECS using Terraform
+description: How to install and run Teleport agents on Amazon ECS
+labels:
+ - platform-wide
+ - reference
+---
+
+Using Amazon ECS and Terraform allows you to easily deploy and manage Teleport Agents that are closer to your AWS resources.
+
+This flow is ideal when you want to automatically discover and secure access to your EKS clusters, RDS databases or other supported AWS resources.
+
+## How it works
+Using Terraform, a Teleport Agent is deployed and kept up to date in Amazon ECS.
```

Comment:
```suggestion

A Terraform module deploys a Teleport Agent on Amazon ECS and keeps it up to date.
```
Using active voice instead of passive voice.